src/Encryptor/LegacyEncryptor.php

```diff
@@ -5,32 +5,24 @@
 use Carnage\EncryptedColumn\Exception\PopArtPenguinException;
 use Carnage\EncryptedColumn\ValueObject\EncryptorIdentity;
 use Carnage\EncryptedColumn\ValueObject\IdentityInterface;
+use Carnage\EncryptedColumn\ValueObject\Key;
 use phpseclib\Crypt\Base;
 use phpseclib\Crypt\Rijndael;
 
 class LegacyEncryptor implements EncryptorInterface
```

Comment:
Might be some nitpicking here but since this class only decrypts perhaps a better name would be `LegacyDecryptor`? Might make the non-working encryption behavior more obvious. Not in the scope of this PR, I know, but thought I'd mention it.